src/couch_pse_tests/src/cpse_test_read_write_docs.erl

```diff
@@ -0,0 +1,318 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(cpse_test_read_write_docs).
+-compile(export_all).
+
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("couch/include/couch_db.hrl").
+
+
+setup_each() ->
+    {ok, Db} = cpse_util:create_db(),
+    Db.
+
+
+teardown_each(Db) ->
+    ok = couch_server:delete(couch_db:name(Db), []).
+
+
+cpse_read_empty_docs(Db) ->
```

Comment:
Name of the test is misleading. For some reason I read the test name as "Read documents which are empty". It could be just me though.
`cpse_read_missing_docs` or `cpse_read_non_existent_docs` would be better IMO.

Reply:
`cpse_read_docs_from_empty_db` is also an option.

Reply:
Renamed to `cpse_read_docs_from_empty_db/1`